src/Home/RankingItem/index.tsx

```diff
@@ -0,0 +1,31 @@
+import styled from '@emotion/styled';
+import { useState } from 'react';
+
+import AgeTabs from './AgeTabs';
+import ProductList from './ProductList';
+import TypeTabs from './TypeTabs';
+
+const GiftRankingPage = () => {
+  const [activeCategory, setActiveCategory] = useState<string>('all');
```

Comment:
string 타입이 아니라 해당 카테고리가 가질 수 있는 카테고리로 type 을 지정해주는게 추론 가능한 타입을 좁힌다는 측면에서 좋습니다.

```suggestion
  const [activeCategory, setActiveCategory] = useState<Category>('all');
```